guides/common/modules/proc_deleting-multiple-content-view-versions.adoc

```diff
@@ -0,0 +1,18 @@
+[id="Deleting_Multiple_Content_View_Versions_{context}"]
+= Deleting Multiple Content View Versions
+
+You can delete multiple content view versions simultaneously.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Content View*.
+. Click the content view you want to delete versions of.
+. Navigate to the *Versions* tab.
+. Select the check box of the version or versions you want to delete. 
+. Click the three dots icon at the top of the list of content views.
+. Click *Delete*.
+. Clicking delete opens the deletion wizard.
+. The deletion wizard displays any affected environments.
+. If there are no affected environments, review the details and click *Delete*. 
+. If there are any affected environments, the deletion wizard has the option to reassign any hosts or activation keys before deletion.
+. Review the details of the actions that will be performed upon deletion of these content view versions.
```

Comment:
I'm not sure if we capitalize Content View Versions... @maximiliankolb Can you please weigh in?

Reply:
Looking at existing docs, you cannot go wrong here:
```

foreman-documentation git:(master) git grep "Content View Version" | wc
      13     166    2133
foreman-documentation git:(master) git grep "Content View version" | wc
      31     566    6125
```

I don't really have a preference, do you?

If we come to a decision, I just might fix that real quick.

Reply:
Okay, I'd go with "Content View version", because there are more occurences, so it's like the more established convention. How does that sound?